doc/about.rst

```diff
@@ -91,6 +93,16 @@ Environment also funds several students to work on the project part-time.
 .. image:: images/nyu_short_color.png
    :width: 200pt
    :align: center
+   :target: http://cds.nyu.edu/mooresloan/
+
+`Telecom Paristech <http://www.telecom-paristech.com>`_ funds Manoj Kumar (2014),
+Tom Dupré la Tour (2015), Thierry Guillemot (2016) and Raghav R V (2016) to
+work on scikit-learn.
+
+.. image:: http://f.hypotheses.org/wp-content/blogs.dir/331/files/2011/03/Logo-TPT.jpg
```

Comment:
It might be good to add this image to our git. I worry that the URL given here will not last forever.